### ISSUE TITLE
Link to git remote repo

### ISSUE THREAD

**Issue by Gaszc** [osxc/legacy-boot#13]
It seems like that you changed the repo name from xc-common/xc-custom to legacy-common/legacy-custom.
In this way, $CM_REPO and $CS_REPO should be changed as well.
or:
## Custom tools clone

Cloning into '/Users/zhangchen/src/github.com/Michaelizm/xc-custom'...
Username for 'https://github.com': Michaelizm
Password for 'https://Michaelizm@github.com':
remote: Repository not found.
fatal: repository 'https://github.com/Michaelizm/xc-custom.git/' not found

Thanks.

**Comment by Gaszc** [comment]
I did clicked the bottom from https://osxc.github.io/: "Fork it!", but what I really forked is legacy-custom instead of xc-custom which is hardcode in script.

**Comment by ninjabong** [comment]
Good catch, the website reflects the original directions for getting up and running with osxc but then @rricard changed things up which seem to only be documented in the starter repos README.md.  I'll take a look at updating the site and legacy-common documentation this weekend when I have some time.

Thanks!
